.swift-version

```diff
@@ -1 +1 @@
-4.1
+4.2
```

Comment:
I think this needs to be locked to a snapshot for now. Can you try changing this line to:

```
swift-4.2-DEVELOPMENT-SNAPSHOT-2018-08-03
```

Reply:
Nope, still fails:

> We don't have build instructions for 4.2-DEVELOPMENT-SNAPSHOT-2018-08-03.